install-pikrellcam.sh

```diff
@@ -313,6 +318,9 @@ sudo sed -i "s|PIKRELLCAM_WWW|$PWD/www|; \
 			s/PORT/$PORT/" \
 			/etc/nginx/sites-available/pikrellcam
 
+if ((DEB_VERSION >= BULLSEYE))
+then
+	sudo sed -i "s/php5/php\/php7.4/" /etc/nginx/sites-available/pikrellcam
 if ((DEB_VERSION >= BUSTER))
```

Comment:
This line must be changed to `elif ((DEB_VERSION >= BUSTER))` otherwise the bash interpreter colapses and the installation does finish with wierd error abuot unexpected end of file.